test/compilable/casmmangling.d

```diff
@@ -0,0 +1,7 @@
+//https://issues.dlang.org/show_bug.cgi?id=23012
+import imports.asmmerge; //.c for reference
+
+static assert(fun22.mangleof == "test1");
+
+static assert(xs.mangleof == "test2");
+
```

Comment:
Apparently blank lines at eof are no longer permitted.

Reply:
It's reasonable to improve consistency among the rest of the files. If you setup the `pre-commit` hook it will do it for you when committing new changes.

Reply:
> How do you setup the `pre-commit` hook?

You download `pre-commit` via your package manager (if not available in your distro package manager you can use `pip`) and do `pre-commit install` inside the repo and that's it. Every time you do a commit it will do pre-commit checks.

This will add a hook into `.git/hooks/pre-commit`. If you already have a custom `pre-commit` hook, you can modify the bash script to run `pre-commit` manually with `pre-commit run`. pre-commit is designed to run only on diffed files so it will run pretty quickly, but if you want to for it to run on all files you can do `pre-commit run --all-files`. You can read more about it here: https://pre-commit.com/ .

Reply:
Fixed